test/jdk/javax/swing/JInternalFrame/8146321/JInternalFrameIconTest.java

```diff
@@ -208,6 +216,8 @@ private static void getIconBufferedImage() throws Exception {
     private static void testIfSame(final String lookAndFeelString)
             throws Exception {
         if (!bufferedImagesEqual(imageIconImage, iconImage)) {
+            ImageIO.write(imageIconImage, "png", new File("imageicon-fail.png"));
+            ImageIO.write(iconImage, "png", new File("iconImage-fail.png"));
```

Comment:
Is this for debugging purpose when the test fails ?

Reply:
yes